File: qlty-cli/tests/lang/java/basic.in/functions/MethodsWithParams.java
class MethodsWithParams {
  private String bar;
  private String baz;

  public MethodsWithParams() {
    this.bar = "";
    this.baz = "";
  }

  public String doSomething(String baz, String bar) {
    this.bar = bar;
    this.baz = baz;
    return this.bar + this.baz;
  }
}

File: qlty-cli/tests/lang/swift/basic.in/functions/MethodsWithParams.swift
// Methods with parameters

class MethodsWithParams {
    private var bar: String
    private var baz: String
    
    init() {
        self.bar = ""
        self.baz = ""
    }
    
    func doSomething(baz: String, bar: String) -> String {
        self.bar = bar
        self.baz = baz
        return self.bar + self.baz
    }
    
    // Method with default parameter values
    func doSomethingElse(baz: String = "default baz", bar: String = "default bar") -> String {
        return baz + bar
    }
    
    // Method with variadic parameters
    func concatenate(strings: String...) -> String {
        return strings.joined()
    }
}